crates/nu-std/tests/test_dt.nu

```diff
@@ -0,0 +1,44 @@
+use std assert
+use std dt *
+
+#[test]
+def equal_times [] {
```

Comment:
could we add
```nushell
dt datetime-diff (date now) (date now)
```
to this test?

it looks to me this one is the real one not working :open_mouth:

Reply:
mm, because it's `later` and then `earlier`, the command above will always be later :sweat_smile: 

could we maybe show the number of nanoseconds to see that the dates are off by only a tiny amount?

Reply:
Modified the error to show the actual difference.  This won't be very interesting if the dates are far apart.

Reply:
:ok_hand: